src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,2105 @@
+        /// <pre>
+        /// The State should be Ready
+        /// </pre>
+        public static void SetPreloadedCommandsFromFile(string filePath)
+        {
+            ErrorCode error = VcMgrSetPreloadedCommandsFromFile(filePath);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "SetPreloadedCommandsFromFile Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Retrieves all available commands.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <returns>
+        /// The Command List else null in case of no System Commands
+        /// </returns>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="InvalidOperationException">This exception can be due to operation failed.</exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
+        /// <pre>
+        /// The State should be Ready
+        /// </pre>
+        public static VoiceCommandList GetCurrentCommands()
+        {
+            IntPtr handle = IntPtr.Zero;
+            ErrorCode error = VcMgrGetCurrentCommands(out handle);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "GetCurrentCommands Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+
+            if (handle == IntPtr.Zero)
+            {
+                Log.Error(LogTag, "GetCurrentCommands handle is null");
+                return null;
+            }
+
+            SafeCommandListHandle list = new SafeCommandListHandle(handle);
+            return new VoiceCommandList(list);
+        }
+
+        /// <summary>
+        /// Sets or Gets the Audio In Type
+        /// the Values of the strings can be "VC_AUDIO_ID_BLUETOOTH" or "VC_AUDIO_ID_MSF"
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="InvalidOperationException">This exception can be due to operation failed.</exception>
```

Comment:
If this exception can be thrown in both cases, please write it as a single instance of `<exception cref="InvalidOperationException">`.